config/agent/vector.yaml

```diff
@@ -0,0 +1,46 @@
+---
```

Comment:
Should I review these default configuration files? They seem orthogonal to the target spec.

Reply:
I included them here as they were included in @binarylogic's original commits - but they're not directly related, no.

Reply:
Ah, yeah, I'd split them out and just leave this PR as introducing the spec.

Reply:
Removed in 0e17884cc69a9f1cd67890cd573bcf1780559227